pom.xml

```diff
@@ -1806,6 +1807,13 @@
       </dependency>
 
       <dependency>
+        <groupId>com.google.testing.compile</groupId>
+        <artifactId>compile-testing</artifactId>
+        <version>${version.com.google.testing.compile}</version>
+        <scope>test</scope>
```

Comment:
Nit picking -- we should not specify `scope` in the `depMgmt` section. This should be done in the module which directly uses the dependency.